resources/serverless/templates/admission_webhook.yaml

```diff
@@ -1,51 +0,0 @@
-{{- $cn := printf "%s-controller-manager" (include "fullname" .) }}
-{{- $ca := genCA (printf "%s-%s" $cn "ca") 3650 }}
-{{- $altName1 := printf "%s.%s" $cn .Release.Namespace }}
-{{- $altName2 := printf "%s.%s.svc" $cn .Release.Namespace }}
-{{- $cert := genSignedCert $cn nil (list $altName1 $altName2) 3650 $ca }}
-apiVersion: v1
-kind: Secret
-metadata:
-  name: {{ template "fullname" . }}-k8s-webhook-cert
-  namespace: {{ .Release.Namespace }}
-  labels:
-    app: {{ template "fullname" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-    release: "{{ .Release.Name }}"
-    heritage: "{{ .Release.Service }}"
-type: Opaque
-data:
-  tls.crt: {{ b64enc $cert.Cert }}
-  tls.key: {{ b64enc $cert.Key }}
----
-apiVersion: admissionregistration.k8s.io/v1
```

Comment:
I think this should be moved the the webhook chart, and add the same labels used in the other webhook configurations.